include/zephyr/dt-bindings/adc/stm32_adc.h

```diff
@@ -84,4 +84,15 @@
 #define FULLY_CONFIGURABLE	1
 /** @} */
 
+/**
+ * @name STM32 ADC oversampler type
+ * This value is to set <st,adc-oversampler>
+ * Refer to the RefMan to know which type corresponds to which ADC.
+ * @{
+ */
+#define OVERSAMPLER_NONE	0	/* No oversampler */
+#define OVERSAMPLER_MINIMAL	1	/* 8 oversampling values available */
+#define OVERSAMPLER_EXTENDED	2	/* 1024 oversampling values available */
+/** @} */
+
```

Comment:
Is this used? it seems this PR decided on using strings for `OVERSAMPLER_` and `_CONFIGURABLE` enums

Reply:
looking a bit further, the defines are used in the driver, not in the devicetree, so I would move these inside the driver as they seemingly don't need to be exposed to the devicetree :)

Reply:
You're right, they're not needed here anymore. Moved into the driver